src/Sarif.UnitTests/Sarif.UnitTests.csproj

```diff
@@ -30,4 +30,31 @@
     <ProjectReference Include="..\Sarif.Converters\Sarif.Converters.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="TestData\InsertOptionalDataVisitor\CoreTests.sarif">
```

Comment:
>None [](start = 5, length = 4)

There's no need to mention the files individually, because you are using `Update`, which [allows]() you to "Modify metadata of a file that was included using a glob". Just:
```
<None Update="TestData\**\*.sarif" CopyToOutputDirectory="true" />
```